.travis.yml

```diff
@@ -14,3 +17,7 @@ deploy:
   gem: kontena-plugin-shell
   on:
     tags: true
+after_deploy:
```

Comment:
Any recommendations?  Just ditch it?  I was nervous of it pushing on feature branches (especially since we are not pushing versioned images here).